### DISCUSSION TITLE
Allow the addition of custom actions for a collection, either singular or on a range of records that hit an endpoint

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13848] [Feature Requests]
Just like dataflows can add a button to trigger it, allowing the addition of a custom action could then trigger a POST request to an endpoint. This could be limited to either the create / edit view, or the collection / index view

**Comment by rijkvanzanten** [comment]
I believe that's what the Manual trigger currently does, but I might be mistaken by what you mean exactly.

<img width="937" alt="CleanShot 2022-06-13 at 11 03 24@2x" src="https://user-images.githubusercontent.com/9141017/173384207-b04a005e-a592-49b0-b3a3-bd34ad9eda60.png">

<img width="409" alt="CleanShot 2022-06-13 at 11 06 15@2x" src="https://user-images.githubusercontent.com/9141017/173384727-ec880f8f-31e3-4081-a068-261cd11d8e82.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
